### ISSUE TITLE
AttributeError: module 'autogen' has no attribute 'Agent'

### ISSUE THREAD

**Issue by hasan0336** [microsoft/autogen#1617]
### Describe the issue

 autogenstudio ui --port 8081
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/bin/autogenstudio", line 5, in <module>
    from autogenstudio.cli import run
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/autogenstudio/__init__.py", line 1, in <module>
    from .chatmanager import *
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/autogenstudio/chatmanager.py", line 6, in <module>
    from .workflowmanager import AutoGenWorkFlowManager
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/autogenstudio/workflowmanager.py", line 9, in <module>
    class AutoGenWorkFlowManager:
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/autogenstudio/workflowmanager.py", line 163, in AutoGenWorkFlowManager
    def load(self, agent_spec: AgentFlowSpec) -> autogen.Agent:
AttributeError: module 'autogen' has no attribute 'Agent'

### Steps to reproduce

 python3 -m venv autogen-yt-demo
 source autogen-yt-demo/bin/activate  
 export OPENAI_API_KEY=xxxxxxxx
 pip install autogenstudio 
 autogenstudio ui --port 8081
 
 My pyhon is 3.11.7

**Comment by victordibia** [comment]
Hi @hasan0336 ,

Thanks for noting this.
What version of autogen do you have? 
The error suggests that `autogen.Agent` does not exist. However it is exported from the autogen [package](https://github.com/microsoft/autogen/blob/80890588ca7af8fdb180c4ee271af44738a1b290/autogen/__init__.py#L4)

Also, can you confirm your autogen version and autogenstudio version. ie share the results of 

- pip show pyautogen 
- pip show autogenstudio

**Comment by 2undvierzig** [comment]
I encountered a similar issue. Upon investigation, I found that both "autogen" and "pyautogen" packages were installed on my system. It turned out that "autogen" is not the official library. To resolve the issue, I removed it using the following command:

`pip uninstall autogen`

Then, I upgraded the "pyautogen" package with this command:

`pip install --upgrade pyautogen`

After these steps, the problem was resolved for me.
